src/content/app-platform/getting-started/index.md

```diff
@@ -0,0 +1,260 @@
+---
+linkTitle: Getting started
+title: Getting started with App Platform
+description: Guide to deploying apps using kubectl and the Giant Swarm Management API.
+weight: 20
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2021-04-21
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - How can I deploy an app using an App CR?
+  - How can I configure an App CR?
+  - How can I see which apps are available using the Management API?
+---
+
+# Getting started with App Platform
+
+## Overview
+
+In this guide we will install an ingress controller which helps you expose your
```

Comment:
In this guide, we will install the NGINX ingress controller app. We'll do this by using kubectl, which creates App CRs using the Kubernetes API of your management cluster.

App CRs can be created this way via your automation or our Web UI.

In general, you can manage App CRs with any tool that can communicate with the Kubernetes API such as Helm or GitOps tools (like Argo CD or Flux).

Reply:
I'm suggesting this flow that for me flows more easily: 

"Here, we use kubectl which uses kubernetes api. this can also be done via automation and web ui. in fact, it can be done using any tool that can communicate with the k8s api!"

Reply:
Yes, that flows better.